### ISSUE TITLE
Great paper and tech...

### ISSUE THREAD

**Issue by jourdain** [seelabutk/tapestry#2]
Your paper was really interesting to read and investigating a stateless server for dynamic rendering definitely seems interesting.

But I wanted to add a couple comments.

You did cover ParaViewWeb/Visualizer+LightViz but missed ArcticViewer which rely on pre-generated images (in some cases) which allow dynamic interaction on the client side without any specific server requirement and therefore will scale without putting any burden on the viewing resource.

You can find the original web site here: https://kitware.github.io/arctic-viewer/

And some usage here: https://openchemistry.github.io/tomvizweb/
The image based one: https://openchemistry.github.io/tomvizweb/docs/images_volume_exploration.html

Anyway great work!

**Comment by auxiliary** [comment]
You're right, we did not see ArcticViewer or TomViz. There's a lot of work that we've planned going forward with Tapestry. In the future, we hope to drive adoption of this idea and wonder what possibilities there are for incorporating ideas from ArcticViewer into this. 

Thank you for your comment. We'd love to work with you to improve scientific visualization on the web.
